code/_onclick/item_attack.dm

```diff
@@ -55,6 +81,19 @@
 		return TRUE
 	return FALSE
 
+/**
+ * Called on an object being right-clicked on by an item
+ *
+ * Arguments:
+ * * obj/item/weapon - The item hitting this atom
+ * * mob/user - The wielder of this item
+ * * params - click params such as alt/shift etc
+ *
+ * See: [/obj/item/proc/melee_attack_chain]
+ */
+/atom/proc/attackby_alt(obj/item/weapon, mob/user, params)
+	return ALT_ATTACK_CALL_NORMAL
```

Comment:
I don't really like this as the default, but there's a toooon of places where right clicking exists now, and I sort of just didn't want to go through all of it. This is for backwards compatibility.

Reply:
Actually fuck it, I'm doing it.

Reply:
Actually, on second thought, this makes more sense than I first thought, a lot of alternate attack code is being used in the middle of attackby's for seemingly good reason.